pointcloud.js

```diff
@@ -455,232 +497,257 @@ proto.update = function(options) {
   var lineColors = options.lineColor
 
   //Picking geometry
-  var pickCounter = 0
+  var pickCounter = -1
```

Comment:
Note: now that the increment for this variable is centralized and move on top of the loop, it was required to initialize it at -1 instead of 0 to keep everything in place.